projects/bouncebit-premium/index.js

```diff
@@ -1,8 +1,23 @@
-const { sumTokensExport } = require('../helper/sumTokens');
+const { sumTokens, sumTokensExport } = require('../helper/sumTokens');
 
 const BBTC = '0xF5e11df1ebCf78b6b6D26E04FF19cD786a1e81dC'
 const BBUSD = '0x77776b40C3d75cb07ce54dEA4b2Fd1D07F865222'
-const stBBTC = '0x7F150c293c97172C75983BD8ac084c187107eA19'
+// const stBBTC = '0x7F150c293c97172C75983BD8ac084c187107eA19'
+
+const stBBTC_STAKE_ABI =
+  "function totalStaked() view returns (uint256)";
+
+async function bouncebitTvl(api, ...args) {
+  const stBBTCStaked = await api.call({  abi: stBBTC_STAKE_ABI, target: '0x7F26aB9263E33de947654F44C5AB439090cfAaf7'})  
+  // stBBTC
+  api.add(BBTC, stBBTCStaked)
```

Comment:
why is staked BTC considered as premium tvl?

Reply:
premium tvl is the amount of TVL as user's participation in cefi custody.
bbscan is the portion of the user's participation in the node pledge defi. In our cedefi, we allow users to participate in cefi with the stBBTC from defi, so that they can get the benefits of cefi and defi at the same time.The user uses STBBTC to participate in Premium, and we will pledge it again without allowing transfer.